aws/templates/id/id_dataset.ftl

```diff
@@ -120,7 +137,11 @@
             "Attributes" : attributes,
             "Roles" : {
                 "Inbound" : {},
-                "Outbound" : {}
+                "Outbound" : {
+                    "default" : "consume",
+                    "produce" : producePolicy!{},
```

Comment:
Think its better to explicitly define producePolicy and consumePolicy as = 
{} at the start of the function.